app/components/back-office-app-services/document-check-function/function-app.tf

```diff
@@ -17,10 +17,10 @@ module "anti_virus_functions" {
   use_app_insights                         = true
 
   app_settings = {
-    CLAM_AV_HOST              = var.clamav_host
-    CLAM_AV_PORT              = "3310"
-    API_HOST                  = var.back_office_api_host
-    DOCUMENT_STORAGE_API_HOST = var.back_office_document_api_host
+    CLAM_AV_HOST                          = var.clamav_host
+    CLAM_AV_PORT                          = "3310"
+    API_HOST                              = var.back_office_api_host
+    pinsstdocsbodevukw001__blobServiceUri = var.back_office_document_api_host
```

Comment:
We don't need this any more, did you rebase from main?

Reply:
this was an old change that was from before you made your changes, let me remove it